.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,16 +15,36 @@
 ### Before
 
 
-
 ### After
 
 
+## Release checklist
+
+[Release process documentation](../doc/how-to/perform-a-release.md)
+
+As part of our continuous deployment strategy we must ensure that this work is ready to be released at any point.
+
+Before merging to `main` we must first confirm:
+
+### Pre-merge checklist
```

Comment:
These `h3`s could be `Pre-merge` and `Post-merge` given the `h2` is `Release checklist` (checklist duplication)